frontend/packages/console-dynamic-plugin-sdk/src/runtime/plugin-dependencies.ts

```diff
@@ -47,16 +47,12 @@ export const resolvePluginDependencies = (
   // eslint-disable-next-line no-console
   console.info(`Resolving dependencies for plugin ${pluginID}`);
 
-  // Include prerelease tag (if present) in the range check
-  // https://github.com/npm/node-semver#prerelease-tags
-  const semverOptions: semver.Options = { includePrerelease: true };
```

Comment:
I agree with this change, but I'm confused why we were seeing different behavior in https://issues.redhat.com/browse/OCPBUGS-1077. Shouldn't that have worked with `includePrerelease` set?

Reply:
@spadgett Sorry for responding late.

[OCPBUGS-1077](https://issues.redhat.com/browse/OCPBUGS-1077) expected that version `4.12.0-0.ci-2022-09-08-121215` matches range `^4.12.0` which is false regardless of `includePrerelease` option.

The [`includePrerelease` option](https://github.com/npm/node-semver#prerelease-tags) in `semver` package only impacts ranges (for the purpose of range matching) that include prerelease tags.

> For example, the range `>1.2.3-alpha.3` would be allowed to match the version `1.2.3-alpha.7`, but it would not be satisfied by `3.4.5-alpha.9`, even though `3.4.5-alpha.9` is technically "greater than" `1.2.3-alpha.3` according to the SemVer sort rules. The version range only accepts prerelease tags on the `1.2.3` version. The version `3.4.5` _would_ satisfy the range, because it does not have a prerelease flag, and `3.4.5` is greater than `1.2.3-alpha.7`.